raft/src/main/java/org/apache/kafka/raft/AppendCommitMode.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.raft;
+
+public enum AppendCommitMode {
```

Comment:
I'm not very happy with the name `commit mode` here, since in raft committing should always be tied with quorum, but `ack mode` seems not very appropriate either. Other recommendation welcomed.

Reply:
I agree that `AckMode` seems reasonable. We have at least one variable in the patch already that is named `ackMode`. At a minimum, we can try to use names consistently.

Reply:
Cool, will change back to ackMode then.